src/GiraffeGenerator.Core/OpenApi.fs

```diff
@@ -159,12 +269,26 @@ type Response =
       MediaType: MediaType
       Kind: TypeKind }
 
+/// Source of binding
+type PayloadLocation =
+    | Body of MediaType
+    // TODO: Cookie
```

Comment:
Let's create issue for this TODO

Reply:
#35. Also referenced it from the comment and from the README